CONTRIBUTORS.md

```diff
@@ -0,0 +1,5 @@
+Cerner Corporation
+
+- Tyler Biethman [@tbiethman]
```

Comment:
should these previous repo contributors be added? https://github.com/cerner/duplicate-package-checker-webpack-plugin/graphs/contributors

Reply:
I can add them